lab-cameron/routes/move-routes.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const createError = require('http-errors');
+const moveCtrl = require('../controllers/move-controller');
+
+module.exports = function(router) {
+  router.get('/move/:id', (req, res) => {
+    if (!req.params.id) return res.status(400).send(createError(400, 'bad request'));
+    moveCtrl.fetchMove(req.params.id)
+    .then(move => {
+      res.json(move);
+    })
+    .catch(err => res.status(400).send(err.message));
+  });
```

Comment:
This module looks good, but again, space your routes out so it's easier to read.